### PR TITLE
Several guidebook changes

### PR DESCRIPTION
**Pull request by DEATHB4DEFEAT**
# Description

- Fix all spelling mistakes (in edited guidebooks)
- Improve Standard Operating Procedure
- Add a ton of cool formatting to things
- Fix inconsistencies in word casing

# Changelog

### CODE REVIEW

**Comment by DEATHB4DEFEAT** [comment]
image

File: Resources/Server Info/SimpleStation14/Guidebook/AlertProcedure/StandardOperatingProcedure.xml [hunk 1]
```diff
@@ -1,63 +1,66 @@
...
-- XenoBlood, or the corpse of a Xenomorph
+- Chloral Hydrate [color=#a4885c]|[/color] Must be prescribed by the Psychologist
+- Lysergic Acid Diethylamide, (LSD) sometimes referred to as 'MindBreaker'
+- Ephedrine [color=#a4885c]|[/color] Must be in the possession of the [color=#52B4E9]Chief Medical Officer[/color] or Security
+- DesoxyEphedrine, commonly referred to as 'Meth'
+- Any explosive or pyrotechnic compound [color=#a4885c]|[/color] Must be in a Science department testing area
+- Any lethal toxin or toxic substance/compound [color=#a4885c]|[/color] Must be in the Medical chemistry lab under the supervision of the [color=#52B4E9]Chief Medical Officer[/color]
+- Any gases that are not normally found in the atmosphere of the station or station's planet [color=#a4885c]|[/color] Must be possessed by the CE or an Atmospheric Technician
+- Liquid Miasma [color=#a4885c]|[/color] Must be possessed by the [color=#52B4E9]Chief Medical Officer[/color] or a Virologist for research purposes only
+- XenoBlood
```
Comment (Pspritechologist) [inline Resources/Server Info/SimpleStation14/Guidebook/AlertProcedure/StandardOperatingProcedure.xml]: Just a splash of colour? :P

File: Resources/Server Info/SimpleStation14/Guidebook/Server/GettingStarted.xml [hunk 2]
```diff
@@ -4,65 +4,64 @@
...
 - Ask your fellow players out of character, either in [color=#66bbff]OOC[/color] or [color=#66e0e0]LOOC[/color].
-- Or of course, ask Administrators via Ahelp (F1) or Discord.
+- Or of course, ask Administrators via [color=#DC143C]AHelp[/color] (F1) or Discord.
 
 
 ## Meta; For The Newbies
 
-Space Station 14 is a role playing game at its core- and Parkstation intends to let that fact truly shine. You play as a member of the station crew, which you can customize from the lobby prior to joining the game.
-From there, you can select the jobs your character preforms, what they look like, how people are expected to refer to them, etc.
+Space Station 14 is a role-playing game at its core- and Parkstation intends to let that fact truly shine. You play as a member of the station crew, which you can customize from the lobby before joining the game.
```
Comment (Pspritechologist) [inline Resources/Server Info/SimpleStation14/Guidebook/Server/GettingStarted.xml]:
```suggestion
Space Station 14 is a role-playing game at its core- and Parkstation intends to let that fact truly shine. You play as a member of the station crew, which you can customize from the lobby prior to joining the game.
```
\>:P

File: Resources/Server Info/SimpleStation14/Guidebook/Server/Parkstation.xml [hunk 3]
```diff
@@ -22,31 +22,32 @@ For now, until stated otherwise, assume something is as it is in Nyano rather th
...
+this means Parkstation is run instead by the Board of Heads, a council of equal-ranking members.
 
 [color=#ff5555]ALL EQUAL RANKING.[/color]
 Heads are expected to all aid in managing the station and to come to decisions among each other, through voting.
-The AI is not technically a Head, but rather serves to aid all of the crew. Being a super computer, they are of course a good source to look to in times of uncertainty, but they outrank no one.
+The [color=#334E6D]AI[/color] is not technically a head but rather serves to aid all of the crew. Being a supercomputer, they are of course a good source to look to in times of uncertainty, but they outrank no one.
 
-The Head of Personnel is unrelated to Service, and instead serves as the only real Command role. They act as an HR of a type, dealing with overall crew managment, IDs, complaints, and communication among the Heads. Note they still outrank no Heads.
+The [color=#334E6D]Head of Personnel[/color] is unrelated to Service and instead serves as the only real Command role. They act as an HR of a type, dealing with overall crew management, IDs, complaints, and communication among the Heads. Note they still outrank no Heads.
```
Comment (Pspritechologist) [inline Resources/Server Info/SimpleStation14/Guidebook/Server/Parkstation.xml]:
```suggestion
The [color=#334E6D]Head of Personnel[/color] is unrelated to Service and instead occupies Command itself. They act as an HR of a type, dealing with overall crew management, IDs, complaints, and communication among the Heads. Note that they still outrank no Heads, and should instead serve as a binding agent more so than a boss.
```
This was poor wording on my part.

**Review by Pspritechologist — CHANGES_REQUESTED** [review]

File: Resources/Server Info/SimpleStation14/Guidebook/AlertProcedure/StandardOperatingProcedure.xml (hunk 1 above)
Comment (Pspritechologist) [reply Resources/Server Info/SimpleStation14/Guidebook/AlertProcedure/StandardOperatingProcedure.xml]: Death you're colouring string.Empty.

**Review by Pspritechologist — APPROVED** [review]

**Pull request merged**